backend/src/apiserver/common/config.go

```diff
@@ -127,3 +131,17 @@ func GetKubeflowUserIDPrefix() string {
 func GetTokenReviewAudience() string {
 	return GetStringConfigWithDefault(TokenReviewAudience, DefaultTokenReviewAudience)
 }
+
+func GetMetadataTLSEnabled() bool {
+	return GetBoolConfigWithDefault(MetadataTLSEnabled, DefaultMetadataTLSEnabled)
+}
+
+func GetCaCertPath() string {
```

Comment:
Could you add a comment indicating that this CA is only used in the driver and launcher? It kind of implies the API server gets configured with this too but it doesn't use it directly.